### PR TITLE
update #

### PR DESCRIPTION
**Pull request by Yingying016**
update Data Prerequisites

### CODE REVIEW

**Review by Yingying016 — COMMENTED** [review]
I updated "#", because this "#" should be aligned with all other "#" in one line. This editing is related to format.

**Review by Yingying016 — COMMENTED** [review]
This line is not neccessary to add, because the oringal file already states that "...If your data is stored at a different location and you can't or don't want to create a symbolic link, you will need to change the data_dir argument...". Therefore this line is deleted.

**Review by Yingying016 — COMMENTED** [review]
I deleted "s" at the end, because the file downloaded is named "basin_mean_forcing" not "basin_mean_forcings".

File: examples/00-Data-Prerequisites/prerequisites.ipynb [hunk 1]
```diff
@@ -55,11 +55,11 @@
...
+    "When training a deep learning model, such as an LSTM, on data from more than one basin it is recommended to also use static catchment attributes as model inputs, alongside the meteorological forcings (see e.g. [this paper](https://hess.copernicus.org/articles/23/5089/2019/)). In tutorial 06, we use the static catchment attributes which contain 7 txt files (i.e. camels_clim.txt, camels_geol.txt, camels_hydro.txt, camels_name.txt, camels_soil.txt, camels_topo.txt, and camels_vege.txt). They can be downloaded from [this page](https://gdex.ucar.edu/dataset/camels/file.html). The downloaded 7 txt files should be saved under a folder called `camels_attributes_v2.0`. Place this folder into the CAMELS US root directory (at the same level of `basin_mean_forcing` and `usgs_streamflow`). So your folder structure should at least look like this:\n",
```
Comment (gauchm) [inline examples/00-Data-Prerequisites/prerequisites.ipynb:58]:
```suggestion
    "When training a deep learning model, such as an LSTM, on data from more than one basin it is recommended to also use static catchment attributes as model inputs, alongside the meteorological forcings (see e.g. [this paper](https://hess.copernicus.org/articles/23/5089/2019/)). In tutorial 06, we use the static catchment attributes that are part of the CAMELS US dataset. They are stored in 7 txt files (camels_clim.txt, camels_geol.txt, camels_hydro.txt, camels_name.txt, camels_soil.txt, camels_topo.txt, and camels_vege.txt) that can be downloaded from [this page](https://gdex.ucar.edu/dataset/camels/file.html). The attribute files should be saved in a folder called `camels_attributes_v2.0`. Place this folder into the CAMELS US root directory (at the same level of `basin_mean_forcing` and `usgs_streamflow`). So your folder structure should at least look like this:\n",
```

**Review by gauchm — COMMENTED** [review]
Hi @Yingying016, thanks for the contribution.
I have added a few changes to the PR and provided a suggestion to rephrase the static attributes download description. Once the suggestion is accepted, we can have a final look at the PR and merge it.

---
For the record, some background on this PR:
The CAMELS website changed, and the data download is now at a different link, and the CAMELS attributes files need to be placed into a separate folder manually.

**Comment by kratzert** [comment]
Closed due to inactivity. Feel free to re-open, once you continue to work on this.

**Pull request closed without merging**